src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -197,9 +222,29 @@ def _on_run_clicked(self) -> None:
         ):
             return
 
+        sequence = self.value()
+
+        # get saving info from the metadata
+        metadata = sequence.metadata.get("pymmcore_widgets", None)
+        save_dir = metadata.get("save_dir") if metadata else None
+        save_name = metadata.get("save_name") if metadata else EXP
+        extension = metadata.get("extension") if metadata else None
+
+        # create the writers path and make sure they are unique
+        writer_path = None
+        if save_dir and extension is not None:
+            path = Path(save_dir) / f"{save_name}"
+            writer_path = ensure_unique(path, extension)
+            self._update_save_name_text(writer_path, extension)
+
         # run the MDA experiment asynchronously
-        self._mmc.run_mda(self.value())
-        return
+        self._mmc.run_mda(sequence, output=writer_path or None)  # type: ignore
```

Comment:
definitely don't type ignore an unexpected keyword argument!  this is alerting you to a very real problem.  you have to either update the requirement pinning, or check the version and provide a fallback

Reply:
Thanks I meant to do it but I forgot